sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -399,6 +411,225 @@ public TaskManagementSteps(
                           .createdBy(tableRow.get(ColumnHeaders.CREATED_BY.toString()))
                           .build()));
         });
+
+    And(
+        "I click Export button in Task Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
+          TimeUnit.SECONDS.sleep(2);
```

Comment:
no sleep needed, please avoid using this when not needed.
There is also a method in helper waiting after element to be clickable